app/views/collections/new.html.erb

```diff
@@ -18,6 +18,11 @@
         <%= text_area_tag 'collection[description]', collection_description_value, :class=>"form-control rich-text-edit" -%>
       </div>
 
+      <%= fields_for(@collection) do |collection_fields| %>
+        <%= render :partial => 'extended_metadata/extended_metadata_type_selection', :locals => {:f => collection_fields,:resource => @collection } %>
+        <%= render :partial => 'extended_metadata/extended_metadata_attribute_input', :locals => {:f => collection_fields, :resource => @collection } %>
+      <% end %>
+
```

Comment:
use the format `partial:` rather than `:partial=>` , and the same for other keys